src/rocm_docs/__init__.py

```diff
@@ -214,7 +214,8 @@ def setup(self) -> None:
                 f"Expected input toc file {toc_in_path} to exist and be"
                 " readable."
             )
-        url, branch = format_toc(self._docs_folder)
+        format_toc(self._docs_folder)
+        url, branch, edit_page = get_branch(self._docs_folder)
```

Comment:
RE: edit_page var included .git (eg: https://github.com/RadeonOpenCompute/rocm-docs-core.git)

Reply:
`re.sub(r".*\.com[/:](.*)\.git", r"\1", remote_url)`

Regular expression is escaping correctly. Edit page works correctly on local builds